frontend/server/src/main/java/com/amazonaws/ml/mms/wlm/WorkLoadManager.java

```diff
@@ -68,14 +68,33 @@ public boolean hasNoWorker(String modelName) {
         return worker.isEmpty();
     }
 
+    public boolean getWorkerStatus(String modelName) {
+        boolean status = true;
```

Comment:
This local variable is not necessary. You can return ture/false in the loop.

Reply:
I would like to keep it this way. I prefer having as little exits from a method as possible.